.travis.yml

```diff
@@ -1,8 +1,9 @@
 language: node_js
 
 node_js:
-  - "4"
-  - "6"
+- '6'
+- '8'
+- '9'
```

Comment:
Try `'10'` next if you get a chance!

Reply:
@ryan-roemer I'll open an issue to add that to all the Victory repos. Can I drop any? Each one takes about 7 minutes to run in CI.

Reply:
6, 8, 10 is a good list. We can optimize the travis build with things like:

- Remove the extra build command since `postinstall` does a full build already
- Switch to `yarn` with a `yarn.lock` file and use `yarn install --frozen-lockfile` for install speed. (The `npm install` is the slowest part of CI).

feel free to ticket out and I can help!